### PR TITLE
[9.0] migration website_mail_snippet_vertical_resize_base module

### PR DESCRIPTION
**Pull request by cubells**
- Updated README
- Updated views
- Updated js

cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Is this finished?

File: website_mail_snippet_vertical_resize_base/views/snippets.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Tecnativa - Jairo Llopis
+     Copyright 2016 Tecnativa - Vicent Cubells
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+
+<odoo>
+
+<template id="snippet_hr"
```
Comment (yajo) [inline website_mail_snippet_vertical_resize_base/views/snippets.xml:8]: This template ID seems misleading, copy-pasted from somewhere, right?

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by rafaelbn** [comment]
Hi @cubells , please could you review comments? Thanks!

**Comment by rafaelbn** [comment]
Please @chienandalu could review this PR? Thanks

**Comment by pedrobaeza** [comment]
This module has been finally dismissed.

**Pull request closed without merging**